src/configure

```diff
@@ -35,37 +35,54 @@ if [ "$os_VENDOR" == "Ubuntu" ] && [ "$#" -gt 2 ]; then
 fi
 
 
-if [ "$os_VENDOR" == "Ubuntu" ] || [ "$os_VENDOR" == "LinuxMint" ]; then
-    ubuntu_ver=${os_RELEASE:0:2}
-    print_title "Build RDM on $os_VENDOR: $ubuntu_ver"         
+if [ "$os_VENDOR" == "Ubuntu" ] || [ "$os_VENDOR" == "LinuxMint" ] || [ "$os_VENDOR" == "elementary" ]; then
+
+    if [ "$os_VENDOR" == "elementary" ]; then
+        ubuntu_ver=${os_RELEASE:0:3}
+    else
+        ubuntu_ver=${os_RELEASE:0:2}
+    fi
+
+    print_title "Build RDM on $os_VENDOR: $ubuntu_ver"
 
     if [ "$os_VENDOR" == "Ubuntu" ] && [ "$ubuntu_ver" == "12" ]; then
         echo "RedisDesktopManager doesn't support Ubuntu 12 since 0.8.8 release."
         exit 1
     fi
-    
-    if [ "$os_VENDOR" == "LinuxMint" ] && [ "$ubuntu_ver" -lt 17 ]; then
-        echo "RedisDesktopManager only supports linux mint >=17."
-        exit 1
+
+    if [ "$os_VENDOR" == "LinuxMint" ]; then
```

Comment:
Consolidate LinuxMint version conversion in the same style as used by elementary below